src/main/java/emu/grasscutter/server/packet/recv/HandlerHitTreeNotify.java

```diff
@@ -0,0 +1,65 @@
+package emu.grasscutter.server.packet.recv;
+
+import java.util.HashMap;
+
+import emu.grasscutter.data.GameData;
+import emu.grasscutter.game.entity.EntityItem;
+import emu.grasscutter.game.player.Player;
+import emu.grasscutter.game.world.Scene;
+import emu.grasscutter.net.packet.Opcodes;
+import emu.grasscutter.net.packet.PacketHandler;
+import emu.grasscutter.net.packet.PacketOpcodes;
+import emu.grasscutter.net.proto.VectorOuterClass;
+import emu.grasscutter.server.game.GameSession;
+import emu.grasscutter.net.proto.HitCollisionOuterClass.HitCollision;
+import emu.grasscutter.utils.Position;
+
+@Opcodes(PacketOpcodes.HitTreeNotify)
+public class HandlerHitTreeNotify extends PacketHandler {
+    private final HashMap<Integer,HashMap<Integer,HitTreeRecord>> hitRecord = new HashMap<>();

```

Comment:
`hitRecord` should be stored as a transient (so it doesnt get saved to db) hashmap in the Player object. That way, you avoid a memory leak when players stop hitting trees and log out.

Reply:
> `hitRecord` should be stored as a transient (so it doesnt get saved to db) hashmap in the Player object. That way, you avoid a memory leak when players stop hitting trees and log out.

I see, so I decide to use `AutoRecycleHashMap`, It will auto recycal key and value

Reply:
> `hitRecord` should be stored as a transient (so it doesnt get saved to db) hashmap in the Player object. That way, you avoid a memory leak when players stop hitting trees and log out.

There may have some problems, please wait my checking

Reply:
> `hitRecord` should be stored as a transient (so it doesnt get saved to db) hashmap in the Player object. That way, you avoid a memory leak when players stop hitting trees and log out.

All done, please review again.